permissions.apib

```diff
@@ -0,0 +1,21 @@
+## Data Structures
+### Permission
+ + id: 93 (number, required)
+ + permission_name: updatePemission (string,required)
+
+### User_Permission
+ + user_permission_id: 63 (number, required)
+ + permission_id: 93 (number, required)
+ + user_id: 42 (number, required)
+
+# Group Permissions
+The Permissions
+
+## Permissions [/api/v1/permission]
+###get Permissionss [GET]
```

Comment:
Is this a typo?

Reply:
are you referring to the double s in "get Permissionss"? then yes. ill fix it in a sec

Reply:
Yes

Reply:
also could you tell which permissions I changed so I can change them back (i'm not sure how to view that here in the pull request page)

Reply:
If you go to the Files Changed tab, it shows the permissions changed for each file as
`100644 → 100755`, where 664 is the previous permissions and 755 are the new permissions

Reply:
thanks for the tip :)